data_generators/user_generators.py

```diff
@@ -1,6 +1,6 @@
 import os
```

Comment:
The name of this file was changed to `generators.py` since it now contains more than just the user generators. This is indicative of not pulling from main recently.